src/docs/asciidoc/reference/manage/environment-intro.adoc

```diff
@@ -0,0 +1,4 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+Your *application* may be deployed and executed in separate zones called *environments*. Each environment is a particular zone that you can organize as you wish. For example, you can have `prod`, `pre-prod`, `dev` environments for the same application.
```

Comment:
For example, you can have `prod`, `pre-prod`, `dev` environments for the same application. **-> For example, you can have one environment based on your application lifecycle: `dev`, `pre-prod`, `prod`**.

We should add a warning for trial accounts to indicate that there is only one environment.